networkx/algorithms/traversal/depth_first_search.py

```diff
@@ -43,6 +48,7 @@ def dfs_edges(G, source=None):
     -----
     Based on http://www.ics.uci.edu/~eppstein/PADS/DFS.py
     by D. Eppstein, July 2004.
+    https://en.wikipedia.org/wiki/Depth-limited_search
```

Comment:
Readers of the rendered documentation should usually never see raw URLs. Instead add a meaningful English sentence with a sensible hyperlink. For example,

    The implementation of this function is adapted from David Eppstein's
    depth-first search function in `PADS`_, with modifications to allow depth
    limits based on the Wikipedia article "`Depth-limited search`_".

    .. _PADS: http://www.ics.uci.edu/~eppstein/PADS
    .. _Depth-limited search: https://en.wikipedia.org/wiki/Depth-limited_search

For more information on docstring format, see the [Sphinx documentation][1].

[1]: www.sphinx-doc.org/en/stable/rest.html